shlr/gdb/include/libgdbr.h

```diff
@@ -30,16 +30,75 @@ typedef struct libgdbr_message_t {
 	uint8_t chk;	/*! Cheksum of the current message read from the packet */
 } libgdbr_message_t;
 
-/*! 
+/*!
+ * Structure that stores features supported
+ */
+
+typedef struct libgdbr_stub_features_t {
+	ssize_t pkt_sz; /* Max packet size */
+	unsigned qXfer_btrace_read : 1;
```

Comment:
that :1 thing is non portable, it is not recommended to use this C feature because of that